GetStream/stream-chat-swift
# Delete most of Swiftlint warnings

# What does this PR do
Clears most Swiftlint warnings across the project. I ran `swiftlint autocorrect` to fix some issues which Swiftlint could fix on it own. I also went through files with documentation and splitted the documentation on newLines so we don't get any `line_length` swiftlint rule (It was faster than figure out some automation thing)
# How to test this
 You can run the project go through it, run tests and see if anything is broken, but it really shouldn't be
# Where to start
 Some place worth a logic check are: 

- `ConfigurationViewController.swift` -> `isEmpty` condition
- `WebSocketProviderMock.swift` -> Possibly if the delegate is explicitly not meant to be weak (I don't see any reason) 
- `SimpleUsersViewController.swift`  -> `super.viewDidLoad`, was this intentional?
-  `CombineSimpleUsersViewController.swift` -> `super.viewDidLoad`, was this intentional?

The rest is just documentation, you can suggest if you want the rest of warnings (4) but I believe they are not important for now. At least now you don't see tons of warnings about `line_length` :)
